src/nyaml/nyaml2nxdl.py

```diff
@@ -829,6 +879,9 @@ def xml_handle_attributes(dct, obj, keyword, value, verbose):
     if keyword_typ:
         elemt_obj.set("type", keyword_typ)
 
+    # handle nameType
+    xml_handle_nametype(keyword, keyword_name, dct, elemt_obj)
+
```

Comment:
The `nameType` is an attribute and it should be handled below L:88, where all attributes are handled. The function might also be minimized a bit.

Reply:
Suggestions for slimming down the implementation of the xml_handle_nametype have been implemented.
On the the point where nameTypes should be handled I think the previous comment was sufficient.